.gitignore

```diff
@@ -40,6 +40,7 @@ build-artifacts
 
 #ignore css output and sourcemaps
 ui/css/output/
+ui/app/css/output/
```

Comment:
these can be safely deleted and should not reappear because ui/app no longer exists

Reply:
Hmm seems the files only shows up for me locally? If I just delete them it should be fine I suppose?

Reply:
yeah should be fine, happened to me a lot during the refactor to remove the folder